python/gtsam/preamble/discrete.h

```diff
@@ -13,4 +13,3 @@
 
 #include <pybind11/stl.h>
 
-PYBIND11_MAKE_OPAQUE(gtsam::DiscreteKeys);
```

Comment:
Make sure all unit tests still work in python

Reply:
Yup they all work!